build.sbt

```diff
@@ -40,7 +40,7 @@ lazy val microservice = (project in file("."))
     Test / fork := false,
     Test / parallelExecution := false,
     Test / unmanagedSourceDirectories += baseDirectory.value / "test" / "common",
-    Test / unmanagedSourceDirectories += baseDirectory.value / "testcommon"
+    Test / unmanagedSourceDirectories += baseDirectory.value / "testcommon",
```

Comment:
I have seen scala style complain about trailing commas though having run `sbt scalastyle` against this PR I get no errors. Nonetheless, I'd be inclined to remove the comma